packages/ui-extensions/docs/surfaces/admin/build-docs.mjs

```diff
@@ -0,0 +1,254 @@
+    htmlWrapper(
+      `<s-section padding="none">${htmlString}</s-section>`,
+      'display: grid; place-items: center; background: #F1F1F1',
+    ),
+  none: (htmlString) => htmlWrapper(htmlString, 'padding: 0'),
+};
+
+const transformJson = async (filePath, isExtensions) => {
+  let jsonData = JSON.parse((await fs.readFile(filePath, 'utf8')).toString());
+
+  jsonData.forEach((entry) => {
+    // Temporary to ensure that isOptional is added to all members
+    if (entry.definitions && entry.isVisualComponent) {
+      entry.definitions.forEach((definition) => {
+        if (definition.typeDefinitions) {
+          Object.values(definition.typeDefinitions).forEach((typeDef) => {
+            if (typeDef.members && Array.isArray(typeDef.members)) {
+              typeDef.members.forEach((member) => {
+                // eslint-disable-next-line no-prototype-builtins
+                if (member.hasOwnProperty('isOptional')) return;
+                member.isOptional = true;
+              });
+            }
+          });
+        }
+      });
+    }
+
+    if (entry.defaultExample?.codeblock?.tabs) {
+      const newTabs = [];
+      entry.defaultExample.codeblock.tabs.forEach((tab) => {
+        if (tab.language !== 'preview') {
+          newTabs.push(tab);
+          return;
+        }
+
+        if (tab.layout && !(tab.layout in templates)) {
+          console.warn(
+            `${entry.name} has a layout of ${tab.layout} which is not a valid template.`,
+          );
+        }
+
+        const previewHTML =
+          tab.layout && tab.layout in templates
+            ? templates[tab.layout](tab.code)
+            : templates.default(tab.code);
+
+        newTabs.push(
+          {code: tab.code, language: 'html'},
+          {code: previewHTML, language: 'preview'},
+        );
+      });
+
+      entry.defaultExample.codeblock.tabs = newTabs;
+    }
+  });
+
+  // Merge the App Bridge docs with the Shopify Dev docs
+  if (!isExtensions && shopifyDevExists) {
+    const shopifyDevDocs = path.join(
+      shopifyDevDBPath,
+      'app_bridge/generated_docs_data.json',
+    );
+    const shopifyDevDocsContent = await fs.readFile(shopifyDevDocs, 'utf8');
+    const shopifyDevDocsDocsParsed = JSON.parse(
+      shopifyDevDocsContent.toString(),
+    );
+
+    const filteredDocs = shopifyDevDocsDocsParsed.filter(
+      (entry) => entry.category !== 'Experimental Components',
+    );
+
+    // Combine arrays with shopify dev docs first, followed by new data
+    jsonData = [...filteredDocs, ...jsonData];
+  }
+
+  await fs.writeFile(filePath, JSON.stringify(jsonData, null, 2));
+};
+
+const generateFiles = async (tsconfig, outputDir, isExtensions = true) => {
+  const scripts = [
+    `yarn tsc --project ${docsRelativePath}/${tsconfig} --moduleResolution node  --target esNext  --module CommonJS`,
+    `yarn generate-docs --input ./${srcRelativePath} --typesInput ./${srcRelativePath} --output ./${outputDir}`,
+  ];
+
+  if (isExtensions) {
```

Comment:
We aren't checking for the args in this file so it will always default to unstable. We need to make it 2025-07-rc

Reply:
We get the version on line 8 https://github.com/Shopify/ui-extensions/blob/82ad833438a0625d8a2fd466e3708463b9abb47a/packages/ui-extensions/docs/surfaces/admin/build-docs.mjs#L8

I tophatted it with `yarn docs:admin 2025-07-rc ` and it worked fine. Let me know if I am missing something.